azureLogIngestion/opentelemetry/nrSpanContext.ts

```diff
@@ -0,0 +1,32 @@
+import { Context } from "@opentelemetry/api/build/src/context/types"
```

Comment:
Another import from a build directory. There may be a @types package we could use instead?

Reply:
It looks like we can pull Context from the `@opentelemetry/api` instead of the build directory. There used to be a @types import, but that's been deprecated in favor of the api.